internal_transfers/actions/tests/e2e/simulateSolverSolution.spec.ts

```diff
@@ -31,28 +42,38 @@ async function TxDataFromHash(txHash: string): Promise<MinimalTxData> {
     logs,
   };
 }
-describe.skip("getInternalImbalance(transaction, simulator)", () => {
+describe.skip("simulateSolverSolution(transaction, simulator)", () => {
   test("throws when no competition found", async () => {
     const txHash =
       "0x08100e7ba81be84ee0bdce43db6640e2f992ec9991a740a689e97d20dea9dafa";
     const transaction = await TxDataFromHash(txHash);
     await expect(
-      getInternalizedImbalance(transaction, simulator)
+      simulateSolverSolution(transaction, simulator)
     ).rejects.toThrow(`No competition found for ${txHash}`);
   });
-  test("runs as expected on legit txHash with no imbalance", async () => {
+  test("runs as expected on legit txHash with no internalization", async () => {
     const transaction = await TxDataFromHash(
       "0x3b2e9675b6d71a34e9b7f4abb4c9e80922be311076fcbb345d7da9d91a05e048"
     );
-    const imbalance = await getInternalizedImbalance(transaction, simulator);
-    expect(imbalance).toEqual([]);
+    const imbalance = await simulateSolverSolution(transaction, simulator);
+    expect(imbalance).toEqual(null);
   });
   test("runs as expected on txHash = 0xca0bbc", async () => {
     const transaction = await TxDataFromHash(
       "0xca0bbc3551a4e44c31a9fbd29f872f921548d33400e28debb07ffdc5c2d82370"
     );
-    const imbalance = await getInternalizedImbalance(transaction, simulator);
-    expect(imbalance).toEqual([
+    const imbalance = await simulateSolverSolution(transaction, simulator);
+    expect(imbalance).toMatchSnapshot();
+  });
+});
+describe.skip("completeComposition(transaction, simulator)", () => {
+  test("runs as expected on txHash = 0xca0bbc", async () => {
+    const transaction = await TxDataFromHash(
+      "0xca0bbc3551a4e44c31a9fbd29f872f921548d33400e28debb07ffdc5c2d82370"
+    );
+    const simData = await simulateSolverSolution(transaction, simulator);
+    expect(simData).toBeDefined();
+    expect(getInternalizedImbalance(simData ?? DefaultSimData)).toEqual([
```

Comment:
Nit: I imagine `simData ?? DefaultSimData` could be `simData!` instead, since you are testing if it's non-null immediately above and I imagine the test stops there if `simData` is null. (Same for all other tests.)

Reply:
Yes thanks! I did not know the syntax for this.

Reply:
Fixed.